x-pack/plugins/security_solution/common/api/detection_engine/prebuilt_rules/model/diff/three_way_diff/three_way_diff.ts

```diff
@@ -19,6 +19,12 @@ import type { ThreeWayMergeOutcome } from './three_way_merge_outcome';
 export const MissingVersion = Symbol('MissingVersion');
 export type MissingVersion = typeof MissingVersion;
 
+export enum ThreeWayDiffConflictResolutionResult {
+  NO = 'NO',
```

Comment:
Should this be something like `NO_CONFLICT`? I guess `NO` makes sense in the `conflict: no` assignment but it does look a bit weird standalone

Reply:
Updated the enum for clarity